edk2toolext/environment/conf_mgmt.py

```diff
@@ -194,7 +194,7 @@ def GetVsInstallPath(vsversion, varname):
             if(path is None):
                 # Not specified...find latest
                 (rc, path) = FindWithVsWhere(vs_version=vsversion)
-                if rc == 0 and path is not None:
+                if rc == 0 and path is not None and len(path) > 0:
```

Comment:
to make better should we call os.path.exists()
https://docs.python.org/3.7/library/os.path.html#os.path.exists

Reply:
fixed